src/main/java/duke/commands/DeadlineCommand.java

```diff
@@ -0,0 +1,70 @@
+package duke.commands;
+
+import duke.Deadlines;
+import duke.DukeException;
+import duke.Storage;
+import duke.TaskList;
+
+/**
+ * A command type that the chatting bot can read.
+ */
+public class DeadlineCommand extends Command {
+
+    private String taskName;
+    private String time;
+
+    /**
+     * The constructor of this class.
+     *
+     * @param taskName
+     * @param time
+     */
+    public DeadlineCommand(String taskName, String time) {
+        this.taskName = taskName;
+        this.time = time;
+    }
+
+    /**
+     * A method that gets the task name.
+     *
+     * @return the task name
+     */
+    public String getTaskName() {
+        return this.taskName;
+    }
+
+    /**
+     * A method that gets the time of the deadline.
+     *
+     * @return the time of the deadline
+     */
+    public String getTime() {
+        return this.time;
+    }
+
+    /**
+     * The method that includes the execution of the command.
+     *
+     * @param list
+     * @param store
```

Comment:
Perhaps these parameters could use a one-liner description as in the example given in the [coding standards](https://se-education.org/guides/conventions/java/intermediate.html#comments)?